src/music21/meter.ts

```diff
@@ -213,6 +213,12 @@ export class TimeSignature extends base.Music21Object {
             if (!beams) {
                 return;
             }
+
+            if (el.duration.quarterLength >= this.beatDuration.quarterLength) {
+                beamsList[i] = undefined;
+                return;
+            }
```

Comment:
`beam.Beams.naiveBeams()` assigns Beams objects to elements of a stream whose duration is <= 8th note (`beamableDurationTypes`). The beatDuration for almost every time signature is greater than an 8th note, meaning that notes with a duration of the beat or greater don't get beams. The exception is 3/8 where the beat (by default) is an 8th note. For `meter.TimeSignature.getBeams()` to return sensible results in 3/8, we need a check to ensure that 8th notes do not get beamed.